core/components/atoms/tooltip/tooltip.js

```diff
@@ -66,7 +66,14 @@ const arrowStyles = {
   `
 }
 
-const StyledTooltip = styled.div`
+const Tooltip = ({ content, ...props }) => (
+  <Tooltip.Wrapper>
+    <Tooltip.Element {...props}>{content}</Tooltip.Element>
+    {props.children}
+  </Tooltip.Wrapper>
```

Comment:
Interesting case!

Over here, the wrapper is the first element inside the tooltip, hence should be `Tooltip.Element` to be consistent with other changes in this PR

However, that's not the actual tooltip 😅

Can we create a new convention here? I propose `Tooltip.Trigger`. (Similar use case applies to dropdown as well)

Reply:
Let's do it 😄